# CBIL Databases
[ErythronDB](https://www.cbil.upenn.edu/ErythronDB/)

[VEuPathDB](https://veupathdb.org/veupathdb/app)

[OrthoMCL DB](https://orthomcl.org/orthomcl/app)

[NIAGADS GenomicsDB](https://www.niagads.org/genomics/)

[GUS/Strategies-WDK](past-projects/GUS.md)

## Past projects
[BetaCell](http://www.betacell.org/)

[EPConDB](past-projects/EPConDB.md)

[EpoDB](http://www.cbil.upenn.edu/EpoDB/)

[GAIA](http://www.cbil.upenn.edu/gaia2/)

[LENS (Linking ESTs and their associated Name Space)](http://www.cbil.upenn.edu/LENS/)

[ParaDBs:Parasite Databases of Clustered ESTs](http://www.cbil.upenn.edu/ParaDBs/)

[plasmoMAP: The functional interactome of Plasmodium](http://www.cbil.upenn.edu/plasmoMAP/)

**RNA Abundance Database (RAD)**
RAD, as originally described, provided a schema and approach to capturing functional genomics data primarily aimed at measuring transcript abundance. An implementation of RAD was set up to provide access to studies by CBIL collaborators as well as show off features of RAD. Due to changes in technology and funding, the RAD website is no longer available (a historical description is provided below). RAD as a schema is still very much active and incorporated in the GUS (Genomics Unified Schema) database system used by CBIL (EuPathDB, Beta Cell Genomics) and others. The schema for RAD can be viewed along with the other GUS namespaces through our Schema Browser.
<p>
The RAD web site was a resource for gene expression studies, storing highly curated MIAME-compliant studies (i.e. experiments) employing a variety of technologies such as filter arrays, 2-channel microarrays, Affymetrix chips, SAGE, MPSS and RT-PCR. Data were available for querying and downloading based on the MGED ontology, publications or genes. Both public and private studies (the latter viewable only by users having appropriate logins and permissions) were available from this website.
<p>
RAD underwent major revamping in 2003 (described in Manduchi et al., Bioinformatics, 2004). This was motivated by the ongoing microarray standards effort by MGED, our experience with microarray studies, and a move to place RAD within an integrated databases system, GUS. The original version of RAD described in Stoeckert et al. (Bioinformatics, 2001) was replaced with the GUS-based version.
<p>
Studies at the RAD web site: RAD was used to display and provide access to studies by our collaborators. Many of these were related to our work as part of the Endocrine Pancreas Consortium and the Beta Cell Biology Consortium. These studies can be found in the Beta Cell Genomics section of the Beta Cell Biology web site. The remaining studies were primarily associated with a cardiovascular biology project in collaboration with Prof. Peter Davies and his group at Penn. To access these studies go to the Cardiovascular Biology page. If you got here while looking for supplemental information for one of these studies, you will find the relevant links on this page.

[Stem Cell Genome Anatomy Projects (SCGAP)](http://www.cbil.upenn.edu/SCGAP/)
