# CBIL Databases
[ErythronDB](https://www.cbil.upenn.edu/ErythronDB/)

[VEuPathDB](https://veupathdb.org/veupathdb/app)

[OrthoMCL DB](https://orthomcl.org/orthomcl/app)

[NIAGADS GenomicsDB](https://www.niagads.org/genomics/)

[GUS](past-projects/GUS.md)

## Past projects
[BetaCell](http://www.betacell.org/)

**EPConDB**
EPConDB was created to support the Endocrine Pancreas Consortium funded by NIDDK to identify novel pancreatic genes through the sequencing and analysis of EST libraries. EPConDB became part of the NIDDK Beta Cell Biology Consortium (BCBC) and reconstituted as Beta Cell Genomics to support the efforts of the BCBC.

One of the main outcomes of the Endocrine Pancreas Consortium was the PancChip for microarray analysis. An associated PromoterChip was also generated. Information on these arrays is available at the CBIL Downloads site (http://www.cbil.upenn.edu/downloads_orig/EPConDB/download/) and at ArrayExpress.

Related publications:
EPConDB: a web resource for gene expression related to pancreatic development, beta-cell function and diabetes.
Mazzarelli JM, Brestelli J, Gorski RK, Liu J, Manduchi E, Pinney DF, Schug J, White P, Kaestner KH, Stoeckert CJ Jr.
Nucleic Acids Res. 2007 Jan;35(Database issue):D751-5. Epub 2006 Oct 27.
PMID: 17071715 PMCID: PMC1781120

Novel genes identified by manual annotation and microarray expression analysis in the pancreas.
Mazzarelli JM, White P, Gorski R, Brestelli J, Pinney DF, Arsenlis A, Katokhin A, Belova O, Bogdanova V, Elisafenko E, Gubina M, Nizolenko L, Perelman P, Puzakov M, Shilov A, Trifonoff V, Vorobjeva N, Kolchanov N, Kaestner KH, Stoeckert CJ Jr.
Genomics. 2006 Dec;88(6):752-61. Epub 2006 May 24.
PMID: 16725306

Transcriptional program of the endocrine pancreas in mice and humans.
Kaestner KH, Lee CS, Scearce LM, Brestelli JE, Arsenlis A, Le PP, Lantz KA, Crabtree J, Pizarro A, Mazzarelli J, Pinney D, Fischer S, Manduchi E, Stoeckert CJ Jr, Gradwohl G, Clifton SW, Brown JR, Inoue H, Cras-MÃ©neur C, Permutt MA.
Diabetes. 2003 Jul;52(7):1604-10.
PMID: 12829622

Functional genomics of the endocrine pancreas: the pancreas clone set and PancChip, new resources for diabetes research.
Scearce LM, Brestelli JE, McWeeney SK, Lee CS, Mazzarelli J, Pinney DF, Pizarro A, Stoeckert CJ Jr, Clifton SW, Permutt MA, Brown J, Melton DA, Kaestner KH.
Diabetes. 2002 Jul;51(7):1997-2004.
PMID: 12086925

ArrayExpress entries for PancChip and PromoterChip
A-CBIL-1 - UPenn Mouse PancChip 2.1: http://www.ebi.ac.uk/arrayexpress/arrays/A-CBIL-1/
A-CBIL-2 - UPenn Mouse PancChip 5.0: http://www.ebi.ac.uk/arrayexpress/arrays/A-CBIL-2/
A-CBIL-3 - UPenn Mouse PancChip 2.1.1: http://www.ebi.ac.uk/arrayexpress/arrays/A-CBIL-3/
A-CBIL-6 - UPenn Mouse PancChip 4.0: http://www.ebi.ac.uk/arrayexpress/arrays/A-CBIL-6/
A-CBIL-7 - UPenn Mouse PancChip 2.0: http://www.ebi.ac.uk/arrayexpress/arrays/A-CBIL-7/
A-CBIL-9 - UPenn Mouse PancChip 6.0: http://www.ebi.ac.uk/arrayexpress/arrays/A-CBIL-9/
A-CBIL-10 - UPenn Mouse PancChip 6.1: http://www.ebi.ac.uk/arrayexpress/arrays/A-CBIL-10/
A-CBIL-14 - UPenn Mouse PromoterChip 3.0: http://www.ebi.ac.uk/arrayexpress/arrays/A-CBIL-14/
A-MEXP-1668 - Upenn Mouse PromoterChip 5A.1: http://www.ebi.ac.uk/arrayexpress/arrays/A-MEXP-1668/
A-MTAB-536 - Upenn Mouse PromoterChip 5B.0: http://www.ebi.ac.uk/arrayexpress/arrays/A-MTAB-536/

[EpoDB](http://www.cbil.upenn.edu/EpoDB/)

[GAIA](http://www.cbil.upenn.edu/gaia2/)

[LENS (Linking ESTs and their associated Name Space)](http://www.cbil.upenn.edu/LENS/)

[ParaDBs:Parasite Databases of Clustered ESTs](http://www.cbil.upenn.edu/ParaDBs/)

[plasmoMAP: The functional interactome of Plasmodium](http://www.cbil.upenn.edu/plasmoMAP/)

**RNA Abundance Database (RAD)**
RAD, as originally described, provided a schema and approach to capturing functional genomics data primarily aimed at measuring transcript abundance. An implementation of RAD was set up to provide access to studies by CBIL collaborators as well as show off features of RAD. Due to changes in technology and funding, the RAD website is no longer available (a historical description is provided below). RAD as a schema is still very much active and incorporated in the GUS (Genomics Unified Schema) database system used by CBIL (EuPathDB, Beta Cell Genomics) and others. The schema for RAD can be viewed along with the other GUS namespaces through our Schema Browser.
<p>
The RAD web site was a resource for gene expression studies, storing highly curated MIAME-compliant studies (i.e. experiments) employing a variety of technologies such as filter arrays, 2-channel microarrays, Affymetrix chips, SAGE, MPSS and RT-PCR. Data were available for querying and downloading based on the MGED ontology, publications or genes. Both public and private studies (the latter viewable only by users having appropriate logins and permissions) were available from this website.
<p>
RAD underwent major revamping in 2003 (described in Manduchi et al., Bioinformatics, 2004). This was motivated by the ongoing microarray standards effort by MGED, our experience with microarray studies, and a move to place RAD within an integrated databases system, GUS. The original version of RAD described in Stoeckert et al. (Bioinformatics, 2001) was replaced with the GUS-based version.
<p>
Studies at the RAD web site: RAD was used to display and provide access to studies by our collaborators. Many of these were related to our work as part of the Endocrine Pancreas Consortium and the Beta Cell Biology Consortium. These studies can be found in the Beta Cell Genomics section of the Beta Cell Biology web site. The remaining studies were primarily associated with a cardiovascular biology project in collaboration with Prof. Peter Davies and his group at Penn. To access these studies go to the Cardiovascular Biology page. If you got here while looking for supplemental information for one of these studies, you will find the relevant links on this page.

[Stem Cell Genome Anatomy Projects (SCGAP)](http://www.cbil.upenn.edu/SCGAP/)
